ci/run_playbooks.yml

```diff
@@ -0,0 +1,34 @@
+- hosts: controller
+  vars:
+    fvt_dir: "{{ ansible_env.HOME}}/{{ zuul.projects['github.com/infrawatch/feature-verification-tests'].src_dir }}"
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
```

Comment:
These are teh values I found in https://logserver.rdoproject.org/53/353/26c81f218260da8f78d0c88a1acc80205ff6a13b/github-check/podified-multinode-edpm-deployment-crc/1573d7a/zuul-info/inventory.yaml, which is a log for podified-multinode-edpm-deployment-crc (parent to your job)

      cifmw_openshift_api: api.crc.testing:6443
      cifmw_openshift_kubeconfig: '{{ ansible_user_dir }}/.crc/machines/crc/kubeconfig'
      cifmw_openshift_password: '123456789'
      cifmw_openshift_skip_tls_verify: true
      cifmw_openshift_user: kubeadmin